### PR TITLE
Hide HTML generation instructions

### PR DESCRIPTION
**Pull request by ralfhandl**
Instructions in document starter packages state

> A customized version of this command line for this Work Product should be included in the "Introduction" section of the accompanying markdown and HTML files.

So keep the section and reword to sound ok for all three files.

Latest info from TC Admin: we can remove the instructions.

### CODE REVIEW

**Review by HeikoTheissen — DISMISSED** [review]

**Review by HeikoTheissen — DISMISSED** [review]

**Review by sthagen — APPROVED** [review]
LGTM.

We all love renumbering of examples 😁

**Comment by ralfhandl** [comment]
> LGTM.
> 
> We all love renumbering of examples 😁

No problem now with Heiko's cross-referencing support 😄

**Comment by ralfhandl** [comment]
> We all love renumbering of examples 😁

Now using automatic example numbers everywhere (I hope) 🙄

File: odata-json-format/10 Media Entity.md [hunk 1]
```diff
@@ -175,7 +175,7 @@ control information MUST be included in a response that represents a
 partial result.
 
 ::: example
-Example 28:
+Example ##ex:
```
Comment (HeikoTheissen) [inline odata-json-format/10 Media Entity.md:178]: We must bring this correction into CSD01.

Comment (ralfhandl) [reply odata-json-format/10 Media Entity.md:178]: Then let's move on this in the next TC call

**Review by HeikoTheissen — DISMISSED** [review]

**Review by sthagen — APPROVED** [review]
Review 42: LGTM.

**Review by HeikoTheissen — APPROVED** [review]

**Pull request merged**